magento/adobe-stock-integration
# #486 : [Styles] "See more" related images link is shifted out of image preview area on tablet

### Description (*)

This PR fixed some styling issue which is caused due to https://github.com/magento/adobe-stock-integration/pull/481 and also fixed the see more styling issue in tablet

### Fixed Issues (if relevant)

1. magento/adobe-stock-integration#486: [Styles] "See more" related images link is shifted out of image preview area on tablet

### Manual testing scenarios (*)

1.Use tablet or adjust the browser to iPad resolution (768X1024)
2. Login to admin panel
3. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
4. Click "Search Adobe Stock" button to open images grid
5. Click on any image to expand image preview
6. Expected Result 
![image](https://user-images.githubusercontent.com/39480008/65758681-1e574f00-e137-11e9-9d89-73bfdedf2eb6.png)
